staging/src/k8s.io/pod-security-admission/admission/admission.go

```diff
@@ -233,12 +233,12 @@ func (a *Admission) ValidateNamespace(ctx context.Context, attrs api.Attributes)
 	}
 	obj, err := attrs.GetObject()
 	if err != nil {
-		klog.FromContext(ctx).Error(err, "failed to decode object")
+		klog.FromContext(ctx).Error(err, "Failed to decode object")
```

Comment:
@pohly is the capitalization actually necessary for anything?

Reply:
It's the convention (https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md#remove-string-formatting-from-log-message), presumably to make log output more readable.